app/Services/Adselect.php

```diff
@@ -33,11 +31,11 @@ class Adselect
     public function __construct($endpointUrl, $debug = false)
     {
         $this->endpointUrl = $endpointUrl;
-        $this->rpcClient = new Client($this->endpointUrl);
-        $this->rpcClient->getHttpClient()->withTimeout(1);
-        if ($debug) {
-            $this->rpcClient->getHttpClient()->withDebug();
-        }
+//        $this->rpcClient = new Client($this->endpointUrl);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)